optax/_src/alias.py

```diff
@@ -175,32 +253,79 @@ def rmsprop(
   )
 
 
-def sgd(learning_rate: ScalarOrSchedule,
-        momentum: Optional[float] = None,
-        nesterov: bool = False) -> GradientTransformation:
+def sgd(
+    learning_rate: ScalarOrSchedule,
+    momentum: Optional[float] = None,
+    nesterov: bool = False) -> GradientTransformation:
+  """A canonical Stochastic Gradient Descent optimiser.
+
+  This implements stochastic gradient descent. It also includes support for
+  momentum, and nesterov acceleration, as these are standard practice when
+  using stochastic gradient descent to train deep neural networks.
+
+  References:
+    [Sutskever et al, 2013](http://proceedings.mlr.press/v28/sutskever13.pdf)
+
+  Args:
+    learning_rate: this is a fixed global scaling factor.
+    momentum: (default `None`), the `decay` rate used by the momentum term,
+      when it is set to `None`, then momentum is not used at all.
+    nesterov (default `False`): whether nesterov momentum is used.
+
+  Returns:
+    A `GradientTransformation`.
+  """
   return combine.chain(
       (transform.trace(decay=momentum, nesterov=nesterov)
        if momentum is not None else transform.identity()),
       _scale_by_learning_rate(learning_rate)
   )
 
 
-def yogi(learning_rate: ScalarOrSchedule,
-         b1: float = 0.9,
-         b2: float = 0.999,
-         eps: float = 1e-3) -> GradientTransformation:
+def yogi(
```

Comment:
@mtthss If you want, I can add a short description and provide the params/reference (Adaptive Methods for Nonconvex Optimization -  https://papers.nips.cc/paper/2018/file/90365351ccc7437a1309dc64e4db32a3-Paper.pdf)